expected/oracle_join.out

```diff
@@ -902,29 +968,36 @@ SELECT t1, t1.ctid FROM shorty t1 CROSS JOIN longy t2 ORDER BY t1.id;
  (1,"fixed chau")           | (4294967295,0)
  (1,"fixed chau")           | (4294967295,0)
  (1,"fixed chau")           | (4294967295,0)
+ (1,"fixed chau")           | (4294967295,0)
+ (3,"a\x1B\x07\r\x7Fb    ") | (4294967295,0)
  (3,"a\x1B\x07\r\x7Fb    ") | (4294967295,0)
  (3,"a\x1B\x07\r\x7Fb    ") | (4294967295,0)
  (3,"a\x1B\x07\r\x7Fb    ") | (4294967295,0)
  (4,"short     ")           | (4294967295,0)
  (4,"short     ")           | (4294967295,0)
  (4,"short     ")           | (4294967295,0)
-(9 rows)
+ (4,"short     ")           | (4294967295,0)
+ (7,"          ")           | (4294967295,0)
+ (7,"          ")           | (4294967295,0)
+ (7,"          ")           | (4294967295,0)
+ (7,"          ")           | (4294967295,0)
+(16 rows)
```

Comment:
There's lots of unnecessary churn in `oracle_join.out`. Should I delete this new row afterwards or just keep this?